package/lib-tensorflow-1.4.0-src-cpu/scripts.linux/install.sh

```diff
@@ -0,0 +1,77 @@
+#! /bin/bash
+
+#
+# Extra installation script.
+#
+# See CK LICENSE.txt for licensing details.
+# See CK COPYRIGHT.txt for copyright details.
+#
+
+# Configure environment variables
+# TF configure.py script will ask for all unset variables
+export CC_OPT_FLAGS="-march=native"
+export TF_NEED_S3=0
+export TF_NEED_GDR=0
+export TF_NEED_GCP=0
+export TF_NEED_HDFS=0
+export TF_NEED_OPENCL=0
+export TF_NEED_JEMALLOC=0
+export TF_NEED_VERBS=0
+export TF_NEED_MKL=0
+export TF_DOWNLOAD_MKL=0
+export TF_NEED_MPI=0
+export TF_CUDA_CLANG=0
+export PYTHON_BIN_PATH=${CK_ENV_COMPILER_PYTHON_FILE}
+export PYTHON_LIB_PATH="$($PYTHON_BIN_PATH -c 'import site; print(site.getsitepackages()[0])')"
+
+if [ "$CK_TF_NEED_CUDA" == "YES" ] ; then  
```

Comment:
I think we should enable CUDA if `CK_TF_NEED_CUDA` is defined **and** its value is not `NO`?

Reply:
Why should we check if `CK_TF_NEED_CUDA` is defined? If it is not defined, it will not be equal to `YES`, obviously, and `TF_NEED_CUDA` will be 0. I think it is expected behaviour.